FND/Law/LegalCapacity.rdf

```diff
@@ -123,6 +127,35 @@ THE SOFTWARE IS PROVIDED &apos;AS IS&apos;, WITHOUT WARRANTY OF ANY KIND, EXPRES
 		</rdfs:subClassOf>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-gao-obj;Aim">
```

Comment:
Seems bizarre to have these in LegalCapacity ontology when they're properties linking Goal and Objectives. Clearly the use of this Legal Capacity property "implements" is the wrong choice or it's in the wrong ontology.